test_scrap.py

```diff
@@ -22,3 +23,11 @@ def test_scraping(self):
         posts = test.scrape_full_post()
         assert posts
 
+
+
+class TestDataBase:
```

Comment:
It would make sense to move this to a separate file test_database

Reply:
oh ... ok nice idea